pyalgomate/brokers/finvasia/broker.py

```diff
@@ -120,7 +120,7 @@ def getHistoricalData(api: NorenApi, exchangeSymbol: str, startTime: datetime.da
 
     logger.info(
         f'Retrieving {interval} timeframe historical data for {exchangeSymbol}')
-    ret = api.get_time_price_series(exchange=exchange, token=finvasia.getToken(exchangeSymbol),
+    ret = api.get_time_price_series(exchange=exchange, token=finvasia.getToken(exchangeSymbol).split("|")[1],
```

Comment:
getToken method takes exchange plus symbol format, I guess the split here is not needed, Can you confirm?

Reply:
Right, getToken method takes exchange plus symbol format.

But token argument in get_time_price_series method requires only token not exchange, so I used split in the return value of getToken method to strip of the exchange.

Reply:
Here Split is used in returned value of getToken method

Reply:
In that case, we are already splitting the incoming symbol, Can you create a variable named symbol and reuse splitStrings

Reply:
splitStrings contain exchange and symbol, but I need only the token.

splitStrings = exchangeSymbol.split('|')
will give [NSE, FINNIFTY] but not token  

finvasia.getToken(exchangeSymbol) will give 'NSE|26037' in case of FINNIFTY where I am using split to get only token 26037

Reply:
should I change it in getToken() method to return only token instead of exchange ?